lib/src/theme/icons.dart

```diff
@@ -9,52 +9,33 @@ import '../utils/named_property.dart';
 class DSFRIcons {
   const DSFRIcons._();
 
-  static const String _fontFamily = 'Remix Icon';
-  static String? get fontFamily => _packageName;
+  static const fontFamily = 'Remix Icon';
   static String? _packageName = 'flutter_dsfr';
   static String? get packageName => _packageName;
 
   @visibleForTesting
   static set packageName(String? value) => _packageName = value;
 
-  static IconData error = IconData(
-    0xECA0,
-    fontFamily: _fontFamily,
-    fontPackage: _packageName,
-  );
-  static IconData success = IconData(
-    0xEB80,
-    fontFamily: _fontFamily,
-    fontPackage: _packageName,
-  );
-  static IconData info = IconData(
-    0xEE58,
-    fontFamily: _fontFamily,
-    fontPackage: _packageName,
-  );
-  static IconData warning = IconData(
-    0xEA20,
-    fontFamily: _fontFamily,
-    fontPackage: _packageName,
-  );
-  static IconData closeFill = IconData(
-    0xEB98,
-    fontFamily: _fontFamily,
-    fontPackage: _packageName,
-  );
-  static IconData flashLightFill = IconData(
-    0xED3C,
-    fontFamily: _fontFamily,
-    fontPackage: _packageName,
-  );
+  static const errorWarningFill = IconData(0xECA0, fontFamily: fontFamily);
+  static const errorWarningLine = IconData(0xECA1, fontFamily: fontFamily);
```

Comment:
I think errorWarning is a ambiguous name (because there is a difference between a warning and an error in the design system). Why just not naming it `errorFill` ?

Reply:
Because it's the icon's name on Remix Icon website, as we are wrapping Remix Icons I think it's necessary to use the same naming conventions.
<img width="306" alt="Capture d’écran 2022-07-11 144611" src="https://user-images.githubusercontent.com/14369698/178267430-ce49d303-06c6-448a-aec3-272401038e1b.png">

Reply:
yeah ok it make sense and will help when searching for icons